build.gradle

```diff
@@ -25,6 +25,7 @@ sourceSets {
             compileClasspath += main.output
             runtimeClasspath += main.output
             srcDir file('src/integration-test/java')
+            exclude '**/MySQLIntegrationTest.java'
```

Comment:
there's some TDVT-specific configuration elsewhere in `build.gradle` too -- should we remove that as well?

Reply:
I left everything as is for TDVT and only removed the part in the evergreen script which was running TDVT.
Just in case we want to run again or we decided to try it with MongoSQL dialect at some point and also because I thought a thorough clean-up would be done when removing Mysql dialect.
I had to disable build the unit test in order to stop the build from executing automatically which is why there is the change in the gradle file and not the evergreen script for the unit tests.
I can certainly clean-up and remove more things right now as opposed to when we removed mysql if you feel like it makes more sense. But I'd rather wait until we remove the dialect if that's Ok with you. So that we can still easily check that it still works until we definitely get rid of it.

Reply:
Waiting SGTM!